mod/node-api/handlers/utils/id.go

```diff
@@ -70,36 +70,37 @@ func SlotFromBlockID[StorageBackendT interface {
 	return storage.GetSlotByBlockRoot(root)
 }
 
-// SlotFromExecutionID returns a slot from the execution number ID.
+// ParentSlotFromTimestampID returns the parent slot corresponding to the
+// timestamp ID.
 //
-// NOTE: `executionID` shares the same semantics as `stateID`, with the
-// modification of being able to query by beacon block <executionNumber>
-// instead of <stateRoot>.
+// NOTE: `timestampID` shares the same semantics as `stateID`, with the
+// modification of being able to query by next block's <timestamp> instead of
+// the current block's <stateRoot>.
 //
-// The <executionNumber> must be prefixed by the 'n', followed by the execution
-// number in decimal notation. For example 'n1722463215' corresponds to
-// the slot with execution number 1722463215. Providing just the string
-// '1722463215' (without the prefix 'n') will query for the beacon block with
-// slot 1722463215.
-func SlotFromExecutionID[StorageBackendT interface {
-	GetSlotByExecutionNumber(executionNumber math.U64) (math.Slot, error)
-}](executionID string, storage StorageBackendT) (math.Slot, error) {
-	if !IsExecutionNumberPrefix(executionID) {
-		return slotFromStateID(executionID)
+// The <timestamp> must be prefixed by the 't', followed by the timestamp
+// in decimal UNIX notation. For example 't1728681738' corresponds to the slot
+// which has the next block with a timestamp of 1728681738. Providing just the
+// string '1728681738' (without the prefix 't') will query for the beacon block
+// for slot 1728681738.
+func ParentSlotFromTimestampID[StorageBackendT interface {
```

Comment:
nit: `ParentSlotFromTimestamp` instead of `ParentSlotFromTimestampID`

Reply:
The reason why timestampID is different from timestamp is explained in the comment above this function

Reply:
follows a similar convention to what the beacon spec has set up for `state_id` and `block_id`, for ex https://ethereum.github.io/beacon-APIs/#/Beacon/getStateFork